ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -607,7 +619,7 @@
 
 (defn- max-date-within-90-days? [{max-date :max-date}]
```

Comment:
Function name will be misleading after this, I suggest renaming it since it's referenced in only one place in the code
```
(defn- max-date-within-evaluation-window? [{max-date :max-date}]
```